src/S3/BucketStyleMiddleware.php

```diff
@@ -15,28 +15,22 @@
 class BucketStyleMiddleware
 {
     private static $exclusions = ['GetBucketLocation' => true];
-    private $bucketEndpoint;
```

Comment:
Not needed because this middleware's sole reason to exist now is to handle bucket endpoints.